Management/src/ActuatorApi/README.md

```diff
@@ -0,0 +1,118 @@
+﻿# Steeltoe Management Sample - Actuators, Administrative Tasks, Metrics and Tracing
+
+ActuatorWeb and ActuatorAPI form an ASP.NET Core-powered sample application that demonstrates how to use several
+Steeltoe libraries on their own and with additional tools.
+
+In order to avoid duplicating a significant amount of content, the [ActuatorWeb Readme](../ActuatorWeb/README.md)
+contains the shared information and this document only holds content unique to ActuatorApi.
+
+## Running locally
+
+Be sure to follow the [ActuatorWeb Readme](../ActuatorWeb/README.md) to the point where that application is running
+before continuing here.
+
+### MySQL
+
+This application depends on a MySQL database. Refer to [Common Tasks](../../../CommonTasks.md#MySQL) to start a server
+that matches the credentials defined in [appsettings.Development.json](./appsettings.Development.json).
+
+### Administrative Tasks (initialize the database)
+
+In order to demonstrate [Steeltoe Management Tasks](https://docs.steeltoe.io/api/v3/management/tasks.html), the database
+schema and its contents are managed as administrative tasks.
+
+1. Apply Entity Framework Core database schema migration scripts:
+
+    ```shell
+    dotnet run --runtask=MigrateDatabase
+    ```
+
+1. Run [`ForecastTask`](./AdminTasks/ForecastTask.cs) to predict weather for the next 7 days:
+
+    ```shell
+    dotnet run --runtask=ForecastWeather
+    ```
+
+    *Optional* - Add forecast data starting from a specific date and/or number of days:
+
+    ```shell
+    dotnet run --runtask=ForecastWeather --fromDate=10/18/2024 --days=30
+    ```
+
+    > [!NOTE]  
+    > For the `fromDate` parameter, use values formatted as `yyyy-dd-MM` or `MM/dd/yyyy`.
+
+    > [!TIP]
+    > The task [`ResetTask`](./AdminTasks/ResetTask.cs) can be used to remove all forecast data:
+    > `dotnet run --runtask=ResetWeather`
+
+1. Use your preferred IDE or `dotnet run` to start the application.
+
+## Running on Tanzu Platform for Cloud Foundry
+
+1. Create a MySQL service instance in an org/space
+
+   ```shell
+   cf target -o your-org -s your-space
+   ```
+
+    * When using VMware MySQL for Tanzu Application Service:
+
+      ```shell
+      cf create-service p.mysql db-small sampleMySqlService
+      ```
+
+    * When using the Cloud Service Broker for Azure:
+
+      ```shell
+      cf create-service csb-azure-mysql small sampleMySqlService
+      ```
+
+    * When using the Cloud Service Broker for GCP:
+
+      ```shell
+      cf create-service csb-google-mysql your-plan sampleMySqlService
+      ```
+
+1. Wait for the service to become ready (you can check with `cf services`)
+1. Run the command `cf push` to deploy from source (you can monitor logs with `cf logs actuator-api-management-sample`)
```

Comment:
This is "Run the `cf push` command to" in all other samples, but was changed here to "Run the command `cf push` to". If the original text is wrong, it should be updated in other samples too. If not, I don't think we should deviate here.